content/members.md

```diff
@@ -1,8 +1,41 @@
 +++
-title = "SONIX connected members"
+title = "SONIX Friends"
+description = "SONIX Friends - The member section"
+keywords = ["sonix friends", "connected networks", "membership", "members"]
 +++
 
-# Members
+# Become one of our SONIX Friends
+
+## What Are SONIX Friends?
+
+At SONIX, we call our members "SONIX Friends." These are the networks, organizations, and individuals who connect to our Internet Exchange Points (IXPs) and support our mission of improving Sweden’s internet connectivity. Being a SONIX Friend is about more than just connecting—it’s about contributing to a better-connected, more sustainable future.
```

Comment:
How about we change this to "who connect to our Internet Exchange Points (IXPs) **or** support our mission of improving Sweden’s internet connectivity"?

Reply:
What about this?

`These are the networks, organizations, and individuals who connect to our Internet Exchange Points (IXPs) or that just supports our mission of improving Sweden’s internet connectivity.`

Reply:
SONIX Friends are ones that contribute more than being a plain member. Like sponsoring or donating for example.